bootstrap-joining-demo/marathon-api-docker/build.sbt

```diff
@@ -1,7 +1,21 @@
 import com.typesafe.sbt.packager.docker._
 
+name := "bootstrap-joining-demo-marathon-api-docker"
+
+version := "0.1.0"
+
+scalaVersion := "2.12.4"
+
 enablePlugins(JavaServerAppPackaging)
 
 version := "1.0"
 
 dockerUsername := sys.env.get("DOCKER_USER")
+
+libraryDependencies ++= Vector(
+  "com.lightbend.akka.management" %% "akka-management-cluster-bootstrap" % "0.10.0",
+
+  "com.lightbend.akka.management" %% "akka-management-cluster-http" % "0.10.0",
+
+  "com.lightbend.akka.discovery" %% "akka-discovery-marathon-api" % "0.10.0"
```

Comment:
I would prefer to have a version variable instead of putting it in every line

Reply:
For now I'm just going to declare it in a local `val`. I'm a bit hesitant to make it depend on the root project version because it increases friction of trying the examples out; people have to `publishLocal` first.